pennylane_lightning_kokkos/src/algorithms/CMakeLists.txt

```diff
@@ -2,7 +2,7 @@ project(lightning_kokkos_algorithms LANGUAGES CXX)
 set(CMAKE_CXX_STANDARD 17)
 
 
-set(Kokkos_ALGORITHM_FILES AdjointDiffKokkos.hpp AdjointDiffKokkos.cpp CACHE INTERNAL "" FORCE)
+set(Kokkos_ALGORITHM_FILES AdjointDiffKokkos.hpp AdjointDiffKokkos.cpp ExpValFunctors.hpp MeasuresFunctors.hpp MeasuresKokkos.cpp MeasuresKokkos.hpp CACHE INTERNAL "" FORCE)
```

Comment:
I am not convinced measurements belong in algorithms, as they are simulation specific. Can we mirror the structure of Lightning for this, and keep them with the statevector?